pynvjitlink/tests/test_pynvjitlink.py

```diff
@@ -188,7 +192,7 @@ def test_get_linked_ptx_from_lto(device_functions_ltoir):
     handle = _nvjitlinklib.create('-arch=sm_75', '-lto', '-ptx')
     _nvjitlinklib.add_data(handle, input_type, data, filename)
     _nvjitlinklib.complete(handle)
-    ptx = _nvjitlinklib.get_linked_ptx(handle)
+    ptx = _nvjitlinklib.get_linked_ptx(handle)  # noqa: F841
```

Comment:
Why is this the right fix, rather than removing the unused local variable?
```suggestion
    _nvjitlinklib.get_linked_ptx(handle)
```